packages/graphql/tests/integration/advanced-filtering.int.test.ts

```diff
@@ -680,6 +680,299 @@ describe("Advanced Filtering", () => {
                 })
             );
         });
+
+        test("should find Movies GT string", async () => {
+            const session = await neo4j.getSession();
+
+            const movieType = generateUniqueType("Movie");
+
+            const typeDefs = `
+                        type ${movieType.name} {
+                            title: String
+                        }
+                    `;
+
+            const neoSchema = new Neo4jGraphQL({
+                config: {
+                    features: {
+                        filters: {
+                            String: {
+                                LT: true,
+                                GT: true,
+                                LTE: true,
+                                GTE: true,
+                            },
+                        },
+                    },
+                },
+                typeDefs,
+            });
```

Comment:
**suggestion:** I think this configuration is a level too deep! What if it could be like this? I think `features` sits nicely next to `plugins`. 

```suggestion
            const neoSchema = new Neo4jGraphQL({
                features: {
                    filters: {
                        String: {
                            LT: true,
                            GT: true,
                            LTE: true,
                            GTE: true,
                        },
                    },
                },
                typeDefs,
            });
```

Reply:
Could it worth remove the features level and let the config be:
```javascript
config: {
  filters: {
      String: {
          LT: true,
          GT: true,
          LTE: true,
          GTE: true
      }
  }
}
```
thoughts?

Reply:
No problem, it makes sense to me as well! I'm going to move this outside the configuration file 😄